examples/generate_coverage_viz.json

```diff
@@ -24,7 +24,7 @@
       "class": "PipelineStepGenerateCoverageViz",
       "module": "idseq_dag.steps.generate_coverage_viz",
       "additional_files": {
-        "info_db": "s3://idseq-database/alignment_data/2018-12-01/nt_info.sqlite3"
+        "info_db": "s3://idseq-database/alignment_data/2018-12-01/nt_info.db"
```

Comment:
why replace? in other steps, we are uploading both sqlite and db files

Reply:
See comment about eliminating sqlite support at runtime the long term.